src/devices/casparCG.ts

```diff
@@ -511,7 +511,7 @@ export class CasparCGDevice extends DeviceWithState<State> implements IDevice {
 				const backgroundStateLayer = backgroundObj ? this.convertObjectToCasparState(mappings, backgroundObj, mapping, false) : undefined
 
 				if (foregroundStateLayer) {
-					channel.layers[mapping.layer] = {
+					channel.layers[layerName] = {
```

Comment:
This change breaks the unit tests, and I don't think it's great idea in general because A) it can trip up casparcg-state and B) if I define layer1 to have { seek: 1 } and layer2 to have { seek: 2 } casparcg-state would probably send commands for both values at the same time which could lead to unexpected result.

Reply:
TV2 use case was addressed in a PR to this branch https://github.com/nrkno/tv-automation-state-timeline-resolver/pull/177